pkg/agent/plugin/workloadattestor/unix/unix.go

```diff
@@ -265,6 +272,7 @@ func (p *Plugin) getGID(proc processInfo) (string, error) {
 func (p *Plugin) getGroupName(gid string) (string, bool) {
 	g, err := p.hooks.lookupGroupByID(gid)
 	if err != nil {
+		p.log.Warn("failed to lookup group name by gid", "gid", gid, "error", err)
```

Comment:
nit: log messages should start with uppercase

Reply:
> Thanks for this @rturner3. Other than the nitpicks on the log casing, I was wondering how you felt about throwing together a few test cases to test the logging? Should hopefully be a quick addition to the PR.

Sure, added some test coverage.